mod/notifications/start.php

```diff
@@ -41,6 +41,10 @@ function notifications_plugin_init() {
 function notifications_page_handler($page) {
 
 	elgg_gatekeeper();
+
+	// Set the context to settings
+	elgg_set_context('settings');
+
```

Comment:
A few lines below this we are doing:
```php
$vars['username'] = $page[1];
```
How about (besides adding the context) we also add the page owner here in the page handler:
```php
$owner = get_user_by_username($page[1]);
elgg_set_page_owner_guid($owner->guid);
```
This way we would get also the rest of the settings links:
 - Edit avatar
 - Edit profile
 - Change your settings
 - Configure your tools
 - Account statistics